packages/design-system/src/components/ChoiceList/Choice.test.tsx

```diff
@@ -50,6 +50,14 @@ describe('Choice', () => {
     expect(el).toBeChecked();
   });
 
+  it('allows for modification of aria attributes', () => {
+    renderChoice({ 'aria-live': 'off', 'aria-relevant': 'none', 'aria-atomic': false });
+    const input = screen.getByRole('checkbox');
+    expect(input).toHaveAttribute('aria-live', 'off');
+    expect(input).toHaveAttribute('aria-relevant', 'none');
+    expect(input).toHaveAttribute('aria-atomic', 'false');
```

Comment:
I actually don't think we want them on both the input and the wrapper. They don't get stripped out of the `inputProps` object, so these custom aria attributes end up on both the wrapper and the input.

Reply:
Sorry confused by this, I see the props on the input only when I pass them, I don't see them on the wrapper.